packages/admin-ui/src/common/types.ts

```diff
@@ -32,6 +32,22 @@ export interface ExposableServiceMapping extends ExposableServiceInfo {
   exposed: boolean;
 }
 
+// Wifi
+
+export interface WifiReport {
+  wifiContainer: PackageContainer;
```

Comment:
Not a fan of sending the wifiContainer data here. Some parts of the UI use `packagesGet` and this uses something else. It can lead to inconsistencies this view says the package is stopped and then packages > WIFI says that the package is running. Just fetch `packagesGet` in the view along with the rest of the `WifiReport` data structure.